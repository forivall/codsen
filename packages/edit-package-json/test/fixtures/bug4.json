{"b": [{"a": [],"o": {}},{"yz": "0"}]}
