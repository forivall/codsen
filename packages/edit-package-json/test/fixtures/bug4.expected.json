{"b": [{"a": [],"o": {}},{"yz": "1"}]}
